frontend/src/queries/nodes/DataTable/ColumnConfigurator/ColumnConfigurator.tsx

```diff
@@ -109,9 +109,7 @@ function ColumnConfiguratorModal({ query }: ColumnConfiguratorProps): JSX.Elemen
             columnKey = column.substring(11)
         }
 
-        if (columnKey.includes('#')) {
-            columnKey = extractExpressionComment(columnKey)
-        }
+        columnKey = columnKey.includes('#') ? extractExpressionComment(columnKey) : trimQuotes(columnKey)
```

Comment:
I realised this is not a new bug, but I don't think this does anything. We now use the SQL `--` comment syntax for comments, at least in `extractExpressionComment`.

Reply:
Didn't know, removed `extractExpressionComment()` from there then